infra/charts/turing/requirements.yaml

```diff
@@ -1,4 +1,14 @@
 dependencies:
   - name: postgresql
-    repository: "https://charts.bitnami.com/bitnami"
+    repository: https://charts.bitnami.com/bitnami
     version: 8.9.8
+  - name: mlp
+    version: 1.0.0
+    repository: file://subcharts/mlp
+    tags:
+      - mlp
+  - name: merlin
+    version: 0.7.0
+    repository: file://subcharts/merlin
+    tags:
+      - mlp
```

Comment:
Why is `vault` not a direct, conditional dependency for Turing? I thought it should be the same as MLP/Merlin.

Reply:
But `vault` is also Merlin's dependency (it's even more Merlin's dependency compared to Turing, because Turing can run without Vault (with the in-cluster credentials provided), but Merlin - can't). So for now Vault is Merlin's dep, but we can consider making it a root-level dependency for MLP once we have that chart updated